docs/modules/docker-compose/attachments/data-transforms/docker-compose.yml

```
@@ -0,0 +1,40 @@
+services:
+  redpanda:
+    image: docker.redpanda.com/redpandadata/redpanda:v${REDPANDA_VERSION:?Set a Redpanda version}
+    command:
+    - redpanda
+    - start
+    - --mode dev-container
+    - --smp 1
+    - --kafka-addr
+    - PLAINTEXT://0.0.0.0:9092,OUTSIDE://0.0.0.0:9093
+    - --advertise-kafka-addr
+    - PLAINTEXT://redpanda:9092,OUTSIDE://localhost:9093
+    ports:
+    - 9093:9093
+    - 8081:8081
+    - 8082:8082
+    - 9644:9644
+    volumes:
+      - ./conf/.bootstrap.yaml:/etc/redpanda/.bootstrap.yaml
+  console:
+    container_name: redpanda-console
+    image: docker.redpanda.com/redpandadata/console:v${REDPANDA_CONSOLE_VERSION:?Set a Redpanda Console version}
```

```suggestion
    image: docker.redpanda.com/redpandadata/console:v${REDPANDA_CONSOLE_VERSION:?Set a Redpanda Console version}
```
Is this correct _Set a Redpanda Console version}_